app/views/assets/_manage_form.html.erb

```diff
@@ -9,10 +10,19 @@
   <%= f.error_messages -%>
 
   <% if show_projects %>
-    <%= render :partial => 'projects/project_selector', :locals => {:resource => resource } %>
+    <%= render partial: 'projects/project_selector', locals: {resource: resource } %>
   <% end %>
   <%= render partial: 'assets/manage_specific_attributes', locals: {f: f, sharing_link: sharing_link} %>
 
-  <%= form_submit_buttons(resource, validate:false) %>
+    <% if params[:is_isa_conform] %>
+    <div class="alert alert-info">
+      <h4>ISA Compliant structure</h4>
+      <p>
+        This <%= "#{t(resource_class)}".downcase.html_safe%> is part of an ISA compliant structure. The permissions of all sample types linked to it will be equally updated.
+      </p>
+    </div>
+    <% end %>
 
-<% end -%>
\ No newline at end of file
+  <%= form_submit_buttons(resource, { validate:false, update_sample_types: true }) %>
```

Comment:
I don't see where `update_sample_types` param is used, and shouldn't be set to true in all cases